
From: William Lee Irwin III <wli@holomorphy.com>

Minor gaffe in sun4d_smp.c for fork_idle(), where the arguments to
init_idle() weren't updated.  This patch updates them to fix it.

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/arch/sparc/kernel/sun4d_smp.c |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN arch/sparc/kernel/sun4d_smp.c~sched-sparc32-fix-fix arch/sparc/kernel/sun4d_smp.c
--- 25/arch/sparc/kernel/sun4d_smp.c~sched-sparc32-fix-fix	2004-08-07 22:11:01.319928888 -0700
+++ 25-akpm/arch/sparc/kernel/sun4d_smp.c	2004-08-07 22:11:01.323928280 -0700
@@ -201,7 +201,7 @@ void __init smp4d_boot_cpus(void)
 			int no;
 
 			/* Cook up an idler for this guy. */
-			p = fork_idle(p, i);
+			p = fork_idle(i);
 			cpucount++;
 			current_set[i] = p->thread_info;
 			for (no = 0; !cpu_find_by_instance(no, NULL, &mid)
_
